test/e2e/specs/editor/blocks/html.spec.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+test.describe( 'HTML block', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+	test( 'can be created by typing "/html"', async ( { editor, page } ) => {
+		// Create a Custom HTML block with the slash shortcut.
+		await page.click( 'role=button[name="Add default block"i]' );
+		await page.keyboard.type( '/html' );
+		await page.locator(
+			`//*[contains(@class, "components-autocomplete__result") and contains(@class, "is-selected") and contains(text(), 'Custom HTML')]`
+		);
```

Comment:
Let's change this to:

```suggestion
		await expect( page.locator( 'role=option[name="Custom HTML"i][selected]' ) ).toBeVisible();
```

Reply:
Done.